owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/LoggingHelper.kt

```diff
@@ -0,0 +1,24 @@
+package com.owncloud.android.lib.common.utils
+
+import info.hannes.timber.FileLoggingTree
+import info.hannes.timber.fileLoggingTree
+import timber.log.Timber
+import java.io.File
+
+object LoggingHelper {
+
+    fun startLogging(directory: File, storagePath: String) {
+        fileLoggingTree()?.let {
+            Timber.forest().drop(Timber.forest().indexOf(it))
+        }
+        if (!directory.exists())
+            directory.mkdirs()
+        Timber.plant(FileLoggingTree(directory, filename = storagePath, delegator = Log_OC::class.java))
```

Comment:
As an idea for https://github.com/owncloud/android/pull/2725#issuecomment-559065677: 
Don't use filename and specify context then the file hast as prefix the package name.